apps/fault/src/main.c

```diff
@@ -166,17 +373,21 @@ static void measure_fault_roundtrip_fn(int argc, char **argv)
         fault();
         SEL4BENCH_READ_CCNT(end);
         results->round_trip[i] = end - start;
+        volatile int j;
+        for (j = 0; j < 10000; j++) {
+
+        }
```

Comment:
This seems to be intended to pause for a bit. Is this something we do by this kind of loop or is there a wait/pause function this should be calling?

Reply:
When I was first running these benchmarks, the variance was very high and it appeared to be a sort of bimodal distribution. I read Shane's thesis on the signal fastpath and he encountered the same issues and found that waiting between runs improved this issue. This is how he did it, but there may be a better standard way which I missed.

Reply:
Happy with the waiting as such, just wanted to check on the way it's done. @kent-mcleod might know more about the standard way this is done in sel4bench/sel4test.